docs/en/connector/flink-sql/Kafka.md

```diff
@@ -0,0 +1,116 @@
+# Flink SQL Kafka Connector
+
+## Description
+
+With kafka connector, we can read data from kafka and write data to kafka using Flink SQL. Refer to the [Kafka connector](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/connectors/table/kafka/) for more details.
+
+
+## Usage
```

Comment:
squash 1~ 5 steps ? I think it's ok

Reply:
> squash 1~ 5 steps ? I think it's ok

Never mind, I think maybe shorter will be better. Because we introduce how to use seatunnel,The knowledge related to kafka is necessary, there is no need to teach others how to configure Kafka in SeaTunnel.

Reply:
got it